scripts/installscripts/buster-install-default.sh

```diff
@@ -567,125 +615,82 @@ main_install() {
         ${apt_get} ${allow_downgrades} install libspotify12 python3-cffi python3-ply python3-pycparser python3-spotify
 
         # Install necessary Python packages
-        cd ${jukebox_dir}/ || exit
-        sudo python3 -m pip install -r requirements-spotify.txt
+        sudo python3 -m pip install -q -r "${jukebox_dir}"/requirements-spotify.txt
     fi
 
-    cd ${jukebox_dir}/misc/sampleconfigs/ || exit
-    sudo rm phoniebox-rfid-reader.service.stretch-default.sample
-    wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/develop/misc/sampleconfigs/phoniebox-rfid-reader.service.stretch-default.sample
-    cd ${jukebox_dir}/scripts/ || exit
-    sudo rm RegisterDevice.py
-    wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/develop/scripts/RegisterDevice.py
-
-    # Jump into the Phoniebox dir
-    cd ${jukebox_dir} || exit
+    local raw_github="https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID"
```

Comment:
#892 has raised the question, why this manual download is necessary, if the repo was already checked out.

Reply:
Since issue #892 was not clearly resolved yet, I did not want to bury the removal of these lines in the clean up effort.

Reply:
I agree. Just leave it like that.